node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -380,11 +383,12 @@ class NodeRuntime(conf: Configuration, host: String, scheduler: Scheduler) {
                         kademliaRPC
                       )
                       .toEffect
-    blockStore = LMDBBlockStore.create[Effect](conf.blockstorage)(
+    blockMap <- Ref.of[Effect, Map[BlockHash, BlockMessage]](Map.empty[BlockHash, BlockMessage])
+    blockStore = InMemBlockStore.create[Effect](
```

Comment:
I think a comment saying this change is temporary until the work by @itegulov is finished would be good.